.travis.yml

```diff
@@ -5,14 +5,11 @@ sudo: false
 
 language: generic
 
-services:
-    - docker
-addons:
-  apt:
-    packages:
-      - docker-ce
```

Comment:
I added this originally to ensure we got the latest Docker. It was part of a desperate attempt to try and fix the gpg/dns/network issues we were having. We don't have that problem anymore, but we might want to track to a newer version of Docker when doing builds.

Reply:
Fair enough but dor now this does nothing but add 20 seconds of checking that the version is up to date which it is. It adds 20 seconds to the build for no reason. Can we re-add it if we ever need it again?